src/main/java/mate/academy/exception/RegistrationException.java

```diff
@@ -0,0 +1,6 @@
+package mate.academy.exception;
+
+public class RegistrationException extends RuntimeException {
+    public RegistrationException(String s) {
```

Comment:
The constructor should pass the message `s` to the superclass constructor by calling `super(s);`. This ensures that the exception message is properly handled and accessible.